lib/worker/index.js

```diff
@@ -8,8 +8,9 @@ const { processPush } = require('../transforms/push');
 const metricsJob = require('./metrics-job');
 const statsd = require('../config/statsd');
 const getRedisInfo = require('../config/redis-info');
+const { getLog } = require('../../lib/config/logger');
+let logger = getLog();
 
```

Comment:
ensure existing test are running fine, you may need to fix them before merging the code

Reply:
Will fix test cases as part of other work item.